src/app/pages/course-add/course-add.template.html

```diff
@@ -1,34 +1,66 @@
-<div class="container">
-    <form class="course-add" (ngSubmit)="createCourse(courseForm)" #courseForm="ngForm">
+<div class="container" *ngIf="formModel">
+    <form class="course-add" (ngSubmit)="saveCourse(courseForm)" #courseForm="ngForm">
         <div class="row">
             <div class="col-sm-4 offset-sm-4">
                 <div class="row">
-                    <h2>Add course</h2>
+                    <h2>
+                        <span *ngIf="this.courseId">Edit </span>
+                        <span *ngIf="!this.courseId">Add </span>
```

Comment:
tutaj mozna jak cos uzyc if / else

https://medium.com/aviabird/ngif-else-lands-in-angular-2-0-a242940e54ff